src/assets/translations/en/main.json

```diff
@@ -1501,7 +1500,6 @@
         "pairExistingDeviceBody": "Connecting a different Ledger device? Click 'Pair new device' to start fresh."
       },
       "failure": {
-        "header": "Error",
```

Comment:
do we need to remove from other language translations, or is that auto updated through the translations gen do you know?

Reply:
@firebomb1 to confirm but I was always told not to touch other translations - assuming this will get autoremoved next translations sync?